app/Console/Commands/ExportEntityTypes.php

```diff
@@ -0,0 +1,563 @@
+
+            $destinationConfig = DB::connection(static::$destination)->getConfig();
+            $this->printConnectionConfig("Destination Connection -> ". static::$destination , $destinationConfig);
+
+            $entityId = $this->option('entity');
+
+            $entity = Entity::find($entityId);
+            if(!$entity){
+                $this->print('Entity not found');
+                return 1;
+            }
+
+            $this->print("Analyze structure starting at base entity: " . $entity->name . " (" . $entity->id . ")");
+            $this->printSectionHeading("Traverse Entity Tree & Extracting Data Models");
+
+            $data = $this->collectAllDataFromEntityChildren($entity);
+
+            $this->printReceipt($data);
+            $this->print("Beginning Transaction");
+
+            DB::connection(static::$destination)->beginTransaction();
+            $user = $this->selectOrCreateTransferUser();
+            $this->transferData($user, $data);
+
+            $this->print("Waiting for user confirmation of this receipt ...");
+            if($this->confirm('Check the receipt carefully before proceeding')) {
+                $this->print("User confirmed receipt. Proceeding with commit");
+                DB::connection(static::$destination)->commit();
+                // This path did not return anything. I return 0 here (see bash return codes)
+                return 0;
+            } else {
+                $this->print("User declined receipt. Rolling back transaction and exiting Program");
+                DB::connection(static::$destination)->rollBack();
+                return 1;
+            }
+        } catch(Exception $e) {
+            $this->print("The program failed unexpectedly:\n" . $e->getMessage() . "\n" . $e->getTraceAsString());
+            return 1;
+        } finally {
+            $this->print("Program finished successfully\nThank you for using this tool\nHave a nice day!");
+            $this->closePrinter();
+            // This path did not return anything. I return 0 here (see bash return codes)
+            return 0;
+        }
+    }
+
+    private function extractThesaurusConcepts($model, array &$allConcepts): void {
+        $collection = ThConcept::where('concept_url', $model->thesaurus_url)->get();
+        foreach($collection as $concept) {
+            $allConcepts[$model->thesaurus_url] = $concept;
+        }
+    }
+
+    private function getLabel($model){
+        $collection = ThConcept::where('concept_url', $model->thesaurus_url)->firstOrFail();
+        return $collection->labels->first()->label;
+    }
+
+    private function collectAllDataFromEntityChildren(Entity $entity): array {
+        $thesaurusLabels = [];
+        $entityTypeIds = $this->getEntityTypes($entity);
+        $entityTypes = array_map(function($entityType) {
+            $entityType = EntityType::findOrFail($entityType);
+            return $entityType;
+        }, $entityTypeIds);
```

Comment:
Simply return `EntityType::...` should be enough. Also eliminates confusion with duplicate variable name